common/src/chain/transaction/account_outpoint.rs

```diff
@@ -41,6 +47,18 @@ impl From<AccountSpending> for AccountType {
 pub enum AccountSpending {
     #[codec(index = 0)]
     Delegation(DelegationId, Amount),
+    // Create certain amount of tokens and add them to circulating supply
+    #[codec(index = 1)]
+    TokenTotalSupply(TokenId, Amount),
+    // Take tokens out of circulation. Not the same as Burn because unmintint means that certain amount
+    // of tokens is no longer supported by underlying fiat currency, which can only be done by
+    // reissuance controller.
+    #[codec(index = 2)]
+    TokenCirculatingSupply(TokenId),
```

Comment:
Why don't we call these TokenMint and TokenUnmint?

Reply:
renamed